src/components/LoadingAnimation.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+
+import LoadingAnimation from './LoadingAnimation'
+
+describe('LoadingAnimation', () => {
+  it('displays loadingspinner gif', () => {
+    const { getByAltText } = render(<LoadingAnimation alt={'loading'} />)
```

Comment:
I don't think you need the curly braces around {'loading'} since it's just a string.

Reply:
Just tested. You are correct, the curly-braces aren't necessary.